### PR TITLE
Update addons_daily dag to call bigquery-etl query

### PR DESCRIPTION
**Pull request by benmiroglio**
The current addons_daily DAG has been turned off since the GCP migration. I've started work to migrate this query from spark to BQ in https://github.com/mozilla/bigquery-etl/pull/819.

I updated the DAG using the [bigquery-airflow cookbook](https://docs.telemetry.mozilla.org/cookbooks/bigquery-airflow.html) as a guide, however I'm not confident that everything looks as it should.

@jklukas Tapping you to take a look since you've seen the bigquery-etl query :).

### CODE REVIEW

**Comment by jklukas** [comment]
This needs to depend on clients_last_seen and search_clients_daily. It might make sense to put this query into the main_summary DAG so it's easier to express the dependency on those two queries. Otherwise, we'll need to add ExternalTaskSensor to wait for the two queries in the other DAG.

**Comment by benmiroglio** [comment]
I added an `ExternalTaskSensor` for those datasets, following the pattern in the [attitudes_daily] DAG.

File: dags/addons_daily.py [hunk 1]
```diff
@@ -1,62 +1,40 @@
...
-        "dthorn@mozilla.com",
-    ],
+    "start_date": datetime.datetime(2020, 3, 16),
+    "email": ["telemetry-alerts@mozilla.com", "bmiroglio@mozilla.com"],
     "email_on_failure": True,
     "email_on_retry": True,
-    "retries": 2,
-    "retry_delay": timedelta(minutes=30),
+    "depends_on_past": False,
+    # If a task fails, retry it once after waiting at least 5 minutes
```
Comment (jklukas) [inline dags/addons_daily.py]:
```suggestion
    # If a task fails, retry it once after waiting at least 30 minutes
```

File: dags/addons_daily.py [hunk 2]
```diff
@@ -1,62 +1,40 @@
...
-    "depends_on_past": True,
-    "start_date": datetime(2019, 6, 10),
-    "email": [
-        "telemetry-alerts@mozilla.com",
-        "bmiroglio@mozilla.com",
-        "smelancon@mozilla.com",
-        "bwright@mozilla.com",
-        "dthorn@mozilla.com",
-    ],
+    "start_date": datetime.datetime(2020, 3, 16),
```
Comment (jklukas) [inline dags/addons_daily.py]: You're missing the import for this.

**Comment by jklukas** [comment]
This looks to be in the right shape now. If you can fix the import and get the tests all passing, we should be ready to merge as soon as https://github.com/mozilla/bigquery-etl/pull/819 is merged.

**Comment by jklukas** [comment]
Looks like we have rebase conflicts here. Can you handle that, @benmiroglio? Let me know if you need help.

**Comment by benmiroglio** [comment]
@jklukas Sorry, I'm not sure where to start here.

**Pull request merged**